cpp/ql/lib/definitions.qll

```diff
@@ -12,8 +12,8 @@ import IDEContextual
  *
  * In some cases it is preferable to modify locations (the
  * `hasLocationInfo()` predicate) so that they are short, and
- * non-overlapping with other locations that might be highlighted in
- * the LGTM interface.
+ * non-overlapping with other locations that might be reported as
+ * code scanning alerts in GitHub.
```

Comment:
Not sure, but "in GitHub" sounds a little like the alerts relate to GitHub. Maybe better to say:
```suggestion
 * code scanning alerts on GitHub.
```

Reply:
great point. I hadn't thought about it that way 👍🏻